src/cli.js

```diff
@@ -1,60 +1,64 @@
 #!/usr/bin/env node
 
 /* eslint-disable id-length, no-process-exit */
+require("regenerator-runtime/runtime")
 const meow = require('meow')
-const readFileStream = require('./readFileStream')
 const { parse, evaluate } = require('groq-js')
 const getStdin = require('get-stdin')
 const chalk = require('chalk')
+const ndjson = require('ndjson')
 require('./gracefulQuit')
 const colorizeJson = require('./colorizeJson')
-const fromUrl = require('./fromUrl')
+const S2A = require('stream-to-async-iterator').default
 const cli = meow(
   `
 Usage
   ${chalk.green(`$ groq '*[<filter>]{<projection>}'`)}
```

Comment:
```suggestion
  ${chalk.green(`$ groq '(@{<projection>}'`)}
  ${chalk.green(`$ groq '(*[<filter>]{<projection>}' --input ndsjon`)}   
```

To highlight two common usages, and the difference between `@` and `*`?